model/aws/registry.yaml

```diff
@@ -12,6 +12,12 @@ groups:
         examples:
           - 79b9da39-b7ae-508a-a6bc-864b2829c622
           - C9ER4AJX75574TDJ
+      - id: aws.extended_request_id
```

Comment:
is this name used by AWS? If not, suggesting `aws.response.id`

Reply:
Yes. The developer guide and JS SDK v3 sample code is shared in description of https://github.com/open-telemetry/semantic-conventions/issues/1670